pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -215,7 +215,9 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
         super.channelInactive(ctx);
         isActive = false;
         log.info("Closed connection from {}", remoteAddress);
-        getBrokerService().getInterceptor().onConnectionClosed(this);
+        if (getBrokerService().getInterceptor() != null) {
+            getBrokerService().getInterceptor().onConnectionClosed(this);
```

Comment:
In BrokerService we are setting this field to null.
We could run into some NPE anyway in the future.
It is better to assign the result of getInterceptor to a local variable and then deference it safely, without calling twice this getter

Reply:
@eolivelli Thanks for your feedback. I will use the a local variable to avoid calling twice of getInterceptor.